packages/next/src/client/components/react-dev-overlay/internal/helpers/group-stack-frames-by-framework.ts

```diff
@@ -1,31 +1,10 @@
 import type { OriginalStackFrame } from './stack-frame'
 
 export type StackFramesGroup = {
-  framework?: 'next' | 'react'
+  framework?: OriginalStackFrame['sourcePackage']
   stackFrames: OriginalStackFrame[]
 }
 
-/**
- * Get the origin framework of the stack frame by package name.
- */
-function getFramework(
-  sourcePackage: string | undefined
-): StackFramesGroup['framework'] {
-  if (!sourcePackage) return undefined
-
-  if (
-    /^(react|react-dom|react-is|react-refresh|react-server-dom-webpack|react-server-dom-turbopack|scheduler)$/.test(
-      sourcePackage
-    )
-  ) {
-    return 'react'
-  } else if (sourcePackage === 'next') {
-    return 'next'
-  }
-
-  return undefined
-}
```

Comment:
Entirely moved to `middleware.ts`. See https://github.com/vercel/next.js/pull/62206/files#diff-128189bb3b9422d7645c072469e295bdcbae2e21ab137294cafb0abe0f75ff79R116-R138